tests/Behat/ElasticsearchContext.php

```diff
@@ -120,7 +120,7 @@ private function loadFixtures(): void
             $bulk = [];
 
             foreach (json_decode($file->getContents(), true) as $document) {
-                if (null === $document['id'] ?? null) {
+                if (null === $document['id']) {
```

Comment:
this is probably wrong, are we sure that the file content contains a list of arrays with `id` property?

If not, should probably be `null === ($document['id'] ?? null)` for readability.